kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/ImageServiceIntegrationTest.java

```diff
@@ -53,20 +40,9 @@ public static void buildAndDeployArtifacts() {
 
     }
 
-    @Override
-    protected KieServicesClient createDefaultClient() throws Exception {
-        KieServicesClient servicesClient = super.createDefaultClient();
-
-        uiServicesClient = servicesClient.getServicesClient(UIServicesClient.class);
-        processClient = servicesClient.getServicesClient(ProcessServicesClient.class);
-
-        return servicesClient;
-    }
-
     @Before
     public void cleanup() {
         cleanupSingletonSessionId();
```

Comment:
This functionality is provided by JbpmKieServerBaseIntegrationTest. It may be removed from here.